scripts/lint/stylelint.ts

```diff
@@ -24,14 +22,14 @@ export const lint = async (entry: WalkEntry, content: string, fix: boolean) => {
           ) ?? "unspecified stylelint error";
 
           log(message, {
-            file,
+            file: relative(REPO_ROOT, entry.path),
             startLine: warning.line,
             endLine: warning.endLine,
             startColumn: warning.column,
             endColumn: warning.endColumn,
             content,
           }, warning.severity);
         });
+        if (result.warnings.length > 0) throw new Error("stylelint error");
```

Comment:
Why would we through here? Wouldn't it not get through all of the files?

Reply:
@nekowinston 👀

Reply:
This is invoked over in
```ts
for await (const entry of stylesheets) {
  lint()
```
and we only throw after all the warnings have been printed.